docs/source/trude.rst

```diff
@@ -1,5 +1,111 @@
 Trude
 ==========
 
-.. caution::
-  This page is currently not created, we are working on it. If you would like to contribute on this documentation, reach `me <helena.almamol@gmail.com>`_!
+*From Germanic, Gertrud, hypocorism: female friend.*
+
+This city produces the light and dark spectrum of SiPMs for dedicated calibration runs. This is achieved by selecting regions in the SiPM waveforms where LED pulses are expected and regions which end 2 microseconds before, respectively and integrating the content within each region. The regions before LED pulses should only contain electronics noise and dark counts giving the zero external light approximation whereas those in time with the pulses will contain one or more detected photoelectrons. The waveform integrals are split into two groups: those with expected photoelectrons (light) and those without expected photoelectrons (dark). Each group produces a different spectrum.
+
+.. _Trude input:
+
+Input
+-----
+
+ * ``/Run/events``
+ * ``/Run/runInfo``
+ * ``/RD/sipmrwf``
+
+.. _Trude output:
+
+Output
+------
+
+* ``/HIST/sipm_dark``: histogram values of the dark spectrum.
+* ``/HIST/sipm_dark_bins``: bin  values of the dark spectrum.
+* ``/HIST/sipm_spe``: histogram values of the light spectrum.
+* ``/HIST/sipm_spe_bins``: bin  values of the light spectrum.
+
+.. _Trude config:
+
+Config
+------
+
+Besides the :ref:`Common arguments to every city`, *Trude* has the following arguments:
+
+.. list-table::
+   :widths: 50 40 120
+   :header-rows: 1
+
+   * - **Parameter**
+     - **Type**
+     - **Description**
+
+   * - ``min|max_bin``
+     - ``float``
+     - Lower/upper limit of the number of :math:`ADCs` of the waveform to be considered for the spectrum.
```

Comment:
Could specify whether this applies to light spectrum or dark spectrum or both

Reply:
Maybe I didn't understand this comment, but Trude is used to compute the spectrum (light and dark) of a given pulsed waveform. Additionally, it can be used to compute just the dark spectrum if the input is a dark waveform, but it's not the main purpose. I don't find it necessary to include this clarification.

Reply:
No worries Miryam, I think it was me who read this part wrong!